examples/Demos/caduceus.scn

```diff
@@ -12,7 +12,7 @@
 
     <DefaultContactManager name="Response" response="FrictionContact" />
 
-    <Camera position="0 30 90" lookAt="0 30 0"/>
+    <InteractiveCamera position="0 30 90" lookAt="0 30 0"/>
```

Comment:
After further investigation, this was set to "Camera" a while back (537a865e1c13103396ced10e86c41985ed8d6375)
I guess this is/was? required by SofaQtQuick. I dont know if it still the case though.

Reply:
I see, let's ping @marques-bruno. 

Maybe once SofaQtQuick is release we could put it back? Because now this camera isn't doing anything with both the QtViewer (not responding to mouse/key events) and QglViewer (simply ignoring all cameras). And since we don't have any other viewer...